src/Workspaces/CSharp/Portable/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -99,14 +99,15 @@ internal sealed class CSharpSyntaxContext : SyntaxContext
             bool isDestructorTypeContext,
             bool isPossibleTupleContext,
             bool isPatternContext,
+            bool isSoftSelectionContext,
             CancellationToken cancellationToken)
             : base(workspace, semanticModel, position, leftToken, targetToken,
                    isTypeContext, isNamespaceContext, isNamespaceDeclarationNameContext,
                    isPreProcessorDirectiveContext,
                    isRightOfDotOrArrowOrColonColon, isStatementContext, isAnyExpressionContext,
                    isAttributeNameContext, isEnumTypeMemberAccessContext, isNameOfContext,
                    isInQuery, isInImportsDirective, IsWithinAsyncMethod(), isPossibleTupleContext,
-                   isPatternContext, cancellationToken)
+                   isPatternContext, isSoftSelectionContext, cancellationToken)
```

Comment:
this really isn't appropriate.  this is putting a Completion concept into the SyntaxContext.  what might be more appropriate here is simply "IsRightSideOfDotAfterNumericType" or something like that.  completion can then interpret that as it wants.

Reply:
(looking below, call this `IsRightSideOfNumericType`).  thanks.